src/main/java/io/fabric8/jenkins/openshiftsync/OpenShiftUtils.java

```diff
@@ -129,6 +140,25 @@ public static String jenkinsJobName(BuildConfig bc) {
     return jenkinsJobName(namespace, name);
   }
 
+  // Since the token we need can change without updating the global config, we need to get it every time we do a call
+  public static void setOpenShiftOauthToken() {
+    GlobalPluginConfiguration config = GlobalConfiguration.all().get(GlobalPluginConfiguration.class);
+    if(config == null){
+       return;
+    }
+    String credentialsId = config.getCredentialsId();
+    List<OpenShiftTokenCredentials> tokens = CredentialsProvider.lookupCredentials(OpenShiftTokenCredentials.class, (ItemGroup)null, ACL.SYSTEM , (DomainRequirement)null);
+    if( tokens != null && tokens.size() > 0 ) {
```

Comment:
not sure you need either of these checks but you definitely don't need the size check.